data/core.toml

```
@@ -16,27 +16,27 @@ advisory = [
 "Serhat Can",
 "Jennifer Davis",
 "Patrick Debois (founder, 2009–2014 lead)",
+"Yvo van Doorn (2021–2023 co-chair)",
 "Nathen Harvey",
 "Bridget Kromhout (2015–2020 lead)",
 "Ken Mugrage",
 "Matty Stratton (2021–2022 co-chair)",
-"Yvo van Doorn (2021–2023 co-chair)"
 ]
 
 emeritus = [
 "Damon Edwards",
 "Bernd Erk",
 "Anthony Goddard",
+"Mine Heck",
 "Lindsay Holmwood",
-"Gildas Le Nadan",
 "Matthew Jones",
+"Gildas Le Nadan",
 "Stephen Nelson-Smith",
+"Mike Rosado"
+"Andrew Clay Shafer",
 "Julian Simpson",
 "Christian Trabold",
 "John Vincent",
 "James Wickett",
-"Andrew Clay Shafer",
 "John Willis",
```

This trailing comma is also an issue

```suggestion
"John Willis"
```

---

I don't think trailing commas matter in TOML?

---

Missing commas definitely matter though, yes ;)